phalcon/cache/backend/file.zep

```diff
@@ -160,6 +173,11 @@ class File extends Backend implements BackendInterface
 					let ret = frontend->afterRetrieve(cachedContent);
 					return ret;
 				}
+			} else {
+				/**
+				 * As the content is expired, deleting the key
+				 */
+				this->delete(keyName);
```

Comment:
I believe that the function has something to return or not return anything at all. But now see at 174 line:
```zephir
return ret;
```